### PR TITLE
Update using-wai-material.md

### PR DESCRIPTION
**Pull request by shawna-slh**

### CODE REVIEW

File: pages/about/using-wai-material.md [hunk 1]
```diff
@@ -92,6 +92,31 @@ Unless otherwise indicated, all WAI material not covered by another license is p
 
 For material under the Creative Commons License, the footer includes the license, such as: "This work is published and licensed under a [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/)."
 
+## Videos
+You may use videos in your presentations, training, etc, if you use the entire video including the ending with the link to w3.org/WAI
```
Comment (iadawn) [inline pages/about/using-wai-material.md:96]: This could be read as either:

* If I use the entire video, I must include the link
* I can only use the entire video and it must include the link

The language is just a wee bit difficult to parse.

File: pages/about/using-wai-material.md [hunk 2]
```diff
@@ -92,6 +92,31 @@ Unless otherwise indicated, all WAI material not covered by another license is p
...
+{:/}
+
+You may create a dubbed version in your language with the following conditions:
+1. You provide the entire video, including the ending with the link to w3.org/WAI
+2. You do an accurate transation. If you would like to change the wording to be smoother in your langauge, you get it approved by W3C staff before recording.
+3. You maintain accessibility:<br>
+   a. You provide text subtitles/captions for your video. They may be "closed captions" that are only shown if turned on.<br>
+   b. If the video has a descriped version, you also provide a translation of the described video.
+4. Optionally, you may include your organization and logo at the beginning and/or end, indicating that you are the translating organization. It must be clear that the video is from W3C. You may _not_ include your logo throughout the video.
+5. You make the video available for others to use. This includes permission for W3C to put it on our website, YouTube channel, and other places.
```
Comment (iadawn) [inline pages/about/using-wai-material.md:117]: Why is this included? There is nothing elsewhere in the permissions that would require modified content be made available for others to use.

File: pages/about/using-wai-material.md [hunk 3]
```diff
@@ -92,6 +92,31 @@ Unless otherwise indicated, all WAI material not covered by another license is p
...
+6. For the original video on the W3C website, you provide a text translation of the current English VTT file. This translation can then be used as the script for the dupping.
```
Comment (iadawn) [inline pages/about/using-wai-material.md:118]: I am not sure what this means or what the intent is?

File: pages/about/using-wai-material.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ permalink: /about/using-wai-material/
 ref: /about/using-wai-material/
 
 footer: >
-  <p><strong>Date:</strong> Updated 1 February 2023.</p>
+  <p><strong>Date:</strong> Updated @@ Month 2024.</p>
   <p><strong>Editor:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
   <p>Developed with the <a href="http://www.w3.org/People/domain?domain=Web+Accessibility+Initiative">WAI staff</a> and <a href="https://www.w3.org/People/functions/comm">W3C Communications staff</a>.</p>
```
Comment (koalie) [inline pages/about/using-wai-material.md:13]: In parallel, please ask w3t-sys to create a WAI Team function since we killed domains the link for WAI no longer works. 
The link for Comm is <https://www.w3.org/staff/comm/>

File: pages/about/using-wai-material.md (hunk 1 above)
Comment (koalie) [reply pages/about/using-wai-material.md:96]: "if" may be a "provided that".
The question remains to be addressed regarding using portions of videos (and if it's not allowed, to state it.)

Comment (shawna-slh) [reply pages/about/using-wai-material.md:96]: We want them to use the entire video, though not sure we want to explicitly say they can't use part. Perhaps we'd want to say they have to use the entire video, unless they get permission from us to use an excerpt with appropriate credit. However, right now I'm thinking that's not worth the extra wording here or effort for us to review!

For now, I've got:
> You may use W3C WAI videos in your presentations, training, etc. Please use the entire video, including the ending with the link to w3.org/WAI

We can revise that later if/when we feel it's important to refine it.

File: pages/about/using-wai-material.md (hunk 2 above)
Comment (shawna-slh) [reply pages/about/using-wai-material.md:105]: because it's just dupping videos. happy to discuss.

File: pages/about/using-wai-material.md (hunk 4 above)
Comment (koalie) [reply pages/about/using-wai-material.md]: Meanwhile, https://www.w3.org/staff/wai/ now exists.
I don't know which is more convenient between editing each occurrence of "http(s)://www.w3.org/People/domain?domain=Web+Accessibility+Initiative" or asking the Systems Team if redirects can be put in place.
